client/src/pages/AddStore.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="honbun">
+    <div>
+      <span style="color: #000000">店名</span>
+      <input
+        v-model="message"
+        type="text"
+        style="
+          color: #000000;
+          background-color: #ffffff;
+          border: 1px solid #b2bbc7;
+          border-radius: 3px;
+        "
+        placeholder="店名を入力"
+      /><br />
+      <span style="color: #000000">説明</span>
+      <input
+        v-model="message"
+        type="text"
+        style="
+          color: #000000;
+          background-color: #ffffff;
+          border: 1px solid #b2bbc7;
+          border-radius: 3px;
+        "
+        placeholder="営業日を入力"
+      /><br />
```

Comment:
こんな感じで、 `<br />` を使うのではなく、内容ごとに div でくくって別の block にまとめておいてほしいです
```suggestion
      <div>
          <span style="color: #000000">店名</span>
          <input
            v-model="message"
            type="text"
            style="
              color: #000000;
              background-color: #ffffff;
              border: 1px solid #b2bbc7;
              border-radius: 3px;
            "
            placeholder="店名を入力"
          />
      </div>
      <div>
          <span style="color: #000000">説明</span>
          <input
            v-model="message"
            type="text"
            style="
              color: #000000;
              background-color: #ffffff;
              border: 1px solid #b2bbc7;
              border-radius: 3px;
            "
            placeholder="営業日を入力"
          />
      </div>
```